skyportal/tests/conftest.py

```diff
@@ -54,31 +56,36 @@
 from skyportal.tests.fixtures import TMP_DIR  # noqa: F401
 from skyportal.models import Obj
 
+
 # Add a "test factory" User so that all factory-generated comments have a
 # proper author, if it doesn't already exist (the user may already be in
 # there if running the test server and running tests individually)
-if (
-    not DBSession()
-    .execute(sa.select(User).filter(User.username == "test factory"))
-    .scalar()
-):
-    DBSession.add(User(username="test factory"))
-    DBSession.commit()
+with warnings.catch_warnings():
+    warnings.simplefilter("ignore", category=SAWarning)
```

Comment:
Why are we suppressing SAWarnings here?

Reply:
Uncaught, this was leading to the error:
```
 skyportal/tests/conftest.py:61: SAWarning: TypeDecorator SlugifiedStr() will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
```
it is not clear how to set `cache_ok` on this query (or even if we want to) so I'm opting to suppress the warning which does not seem to be a real issue (since we're getting the same results as pre sqla 1.4 and the test timing seems similar)

Reply:
After scouring the docs and reading [this thread](https://groups.google.com/g/sqlalchemy/c/zCPZKTxM6b0) I am convinced that setting `cache_ok` to True is fine.  Setting it to False is also OK, but less optimal.  Better than these warning workarounds, I think.

Reply:
Can you propose a fix? I don't know where in this call to set the flag.